### PR TITLE
MM5-6435 Change mail template theme and logo

### PR DESCRIPTION
**Pull request by bbedigi**

### CODE REVIEW

File: DC/6.1.0/mail_templates/standardheader.dcl
[the remark this answers is not in the document]
Comment (bbedigi) [reply DC/6.1.0/mail_templates/standardheader.dcl:1]: We need to overwrite this with the Hub logo on Hubs

[the remark this answers is not in the document]
Comment (TobiasNissen) [reply DC/6.1.0/mail_templates/standardheader.dcl:1]: I will update this PR with these changes when I get to it then.

File: KeyShot/6.1.0/mail_templates/standardheader.dcl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    target = data.mail_template.standardheader
+    body = '
+    <table class="email-wrapper" width="100%" cellpadding="0" cellspacing="0">
+        <tr>
+            <td align="center">
+                <table class="email-content" width="100%" cellpadding="0" cellspacing="0">
+                    <tr>
+                        <td class="email-masthead">
+                            <span class="email-masthead_logo">
+                                <img src="https://static.digizuite.app/keyshot-hub-primary-logo.png" alt="logo" width="215" height="70" />
```
Comment (bbedigi) [inline KeyShot/6.1.0/mail_templates/standardheader.dcl:17]:
```suggestion
                                <img src="https://static.digizuite.app/keyshot-hub-primary-logo.png" alt="logo" width="215" height="61" />
```

Comment (TobiasWolters) [reply KeyShot/6.1.0/mail_templates/standardheader.dcl]: Thanks 🙏

**Comment by TobiasWolters** [comment]
@madjufra Maybe you can have a quick look at this.
We have ensured that the Dock logo is used by default in mail templates.
We then override it in the `KeyShot` layer to ensure that the Hub logo is used in mail templates in the Hub.
I considered using a variable for this, but didn't really like that approach either. Let me know if you think we should approach this differently.

**Comment by madjufra** [comment]
@TobiasWolters Is the actual Logo for MM5 included in another PR?

Other than the fact that I am not overly excited about having two logos (Come on, hub and dock are the same ...), then it looks good

**Comment by TobiasWolters** [comment]
> @TobiasWolters Is the actual Logo for MM5 included in another PR?

Yes, we are handling the MM logo separately 👍 
 
> Other than the fact that I am not overly excited about having two logos (Come on, hub and dock are the same ...), then it looks good

I agree, but well...

**Review by TobiasWolters — APPROVED** [review]

**Pull request merged**